week-4/C-more-complex-objects/exercise-3.js

```diff
@@ -32,18 +32,20 @@ var parkAvenueHouse = {
 
 // returns the full name (first name + last name) of the owner of the house
 function getOwnerFullName(house) {
-
+return house.currentOwner.firstName +" "+ house.currentOwner.lastName;
 }
 
 // returns an array of the owners' email addresses of the two houses
 function getEmailAddresses(house1, house2) {
-
+  return house1.currentOwner.email +", "+ house2.currentOwner.email;
 }
 
 // returns the address for the cheapest house out of the two
 function getCheapestAddress(house1, house2) {
-
-}
+if(house1 < house2){
+  return "Buy " + house1.address;
+} return "Buy " + house2.address;

```

Comment:
That's exactly right! And you don't need an `else` here. Just a minor detail: To make it easier to read your code, make sure to have a line break before the second `return` statement.